### DISCUSSION TITLE
Problem with .canvas files in project

### DISCUSSION THREAD

**Discussion by detourne09** [marcusolsson/obsidian-projects#886] [Help]
Hi

I added some canvases to my project, now when I try to open them I get this error 

Failed to load "_ACTIVE PROJECTS/Digital
Declutter Project/Digital Decluttter
Canvas.canvas". No number after minus sign in JSON at position 1

It seems that adding the canvases to the project has caused a problem.

I have moved the files out of the project folder, created copies etc but just seem unable to open the files, have they been corrupted in some way?

I'd appreciate some help as there was a lot of work on the canvases that now seems lost/inaccesible

Thanks

**Comment by marcusolsson** [comment]
I'm sorry to hear that :( Projects only supports Markdown files. Seems like we need stricter checks so that you can't load other files. I'm not that familiar with the .canvas format, but I believe it's regular JSON if you open it in an editor so you may be able to recover it if you find the corrupted data.

**Comment by detourne09** [comment]
Thanks for replying Marcus

For anyone else with this issue, I fixed this by moving the files out of the project folder, opening in text edit and then removing the lines:
```
---
status: No status
---
```

From the start of the file

**Reply by marcusolsson** [reply]
I'm relieved to hear it was a relatively easy fix 🙏

**Comment by Acylation** [comment]
Sorry for the unintentional written info to other file formats. We've observed some cases and tried to fix it via #842. The lastest public version of Projects, v1.17.3, is expected to not accept non-md file. You can update Projects to eliminate the problem, or if you're currently in this version, then we should consider it a bug. Thanks for posting both the problem and the fix!
